src/v/cloud_topics/level_zero/reader/reader.h

```diff
@@ -20,6 +20,20 @@ class partition;
 namespace experimental::cloud_topics {
 class data_plane_api;
 
+struct level_zero_log_reader_config {
+    static level_zero_log_reader_config
+    from_kafka_reader_config(kafka::log_reader_config cfg) {
+        return level_zero_log_reader_config{.cfg = std::move(cfg)};
+    }
+
+    kafka::log_reader_config cfg;
```

Comment:
nit: maybe call it `kcfg` so that when it is reference as cfg.cfg or config.cfg it is less "abstract". `cfg.kcfg` reads slightly better to me